Sources/core/objects/Interface.cpp

```diff
@@ -86,6 +92,19 @@ Slot<Item>& HumanInterface::GetActiveHand()
         return l_hand_;
 }
 
+void HumanInterface::UpdatePulling(bool isPulling)
+{
+    pulling_ = isPulling;
```

Comment:
Tho midnight ui only have one button and it appears when someone is pulling , so I implemented it that way.